### PR TITLE
Login UI

### PR DESCRIPTION
**Pull request by Trollface8D**

### CODE REVIEW

File: src/app/login/page.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import Link from 'next/link'
 import { signIn } from 'next-auth/react'
 import { useRouter, redirect } from 'next/navigation'
 import { useSession } from 'next-auth/react';
-
+// I love cat
```
Comment (Besutodesuka) [inline src/app/login/page.jsx]: wtf I did not wrote this

Comment (Besutodesuka) [reply src/app/login/page.jsx]: github cooked me 💀

**Review by Besutodesuka — CHANGES_REQUESTED** [review]

File: src/app/login/page.jsx (hunk 1 above)
Comment (Besutodesuka) [reply src/app/login/page.jsx]: why bro give me 2FA feature

Comment (Besutodesuka) [reply src/app/login/page.jsx]: if my friend can just by the password when I commit 💀

Comment (Besutodesuka) [reply src/app/login/page.jsx]: I love cat btw UWU

**Review by Trollface8D — COMMENTED** [review]
I fixed it...

**Review by Besutodesuka — APPROVED** [review]
lgtm

**Pull request merged**